polynote-kernel/src/main/scala/polynote/kernel/remote/transport.scala

```diff
@@ -106,22 +108,20 @@ class SocketTransportServer private (
   } yield ()
 
   override val responses: Stream[TaskB, RemoteResponse] =
-      Stream.eval(Logging.info("Connected. Decoding incoming messages")).drain ++
         channels.mainChannel.bitVectors
-          .interruptWhen(closed.get.either)
+          .interruptAndIgnoreWhen(closed)
           .through(scodec.stream.decode.pipe[TaskB, RemoteResponse])
-          .handleErrorWith {
-            err => Stream.eval(Logging.error("Response stream terminated due to error", err)).drain
-          } ++ Stream.eval(Logging.info("Response stream terminated")).drain
```

Comment:
Did we lose these errors or where they moved somewhere else?

Reply:
The error was never actually caught by `handleErrorWith`. Now it is caught, upstream from here, because everything propagates it when it closes.

The extra logging message at the end was just redundant.